client/src/Plant/PlantList.js

```diff
@@ -0,0 +1,39 @@
+import {useEffect, useState} from 'react';
+import {Link} from 'react-router-dom';
+import Api from '../Api';
+
+function PlantList(){
```

Comment:
Let's rename this file and component to PlantsList (plural Plants). Because a list shows a collection/multiple Plants.